ports/freetds/portfile.cmake

```
@@ -1,12 +1,9 @@
 vcpkg_from_github(

```

```suggestion
vcpkg_fail_port_install(ON_TARGET "uwp" "linux" "osx")

vcpkg_from_github(
```
Maybe We can consider to add this unsupported messages for this port.